designer/server/src/main/scala/pl/touk/nussknacker/ui/api/ProcessesResources.scala

```diff
@@ -104,6 +106,12 @@ class ProcessesResources(
               }
             }
           }
+        } ~ path("statusDefinitions") {
```

Comment:
In the ideal REST world, there should be /statuses/ and StatusesResource.. But Let's leave it.

Reply:
Lets make a step towards ideal world ;) Moved this path to `StatusResources`